build.go

```diff
@@ -82,9 +100,8 @@ func Build(entries EntryResolver, dependencies DependencyManager, environment En
 			return packit.BuildResult{}, err
 		}
 
-		nodeLayer.Launch = entry.Metadata["launch"] == true
-		nodeLayer.Build = entry.Metadata["build"] == true
-		nodeLayer.Cache = entry.Metadata["build"] == true
+		nodeLayer.Launch, nodeLayer.Build = entries.MergeLayerTypes("node", context.Plan.Entries)
```

Comment:
Hmm. I'm not sure. Maybe a question for @ForestEckhardt. I guess it does hide away some of the layer metadata logic, but it's also nice because it just something super redundant across all of our buildpacks.